server/src/test/java/org/opensearch/rest/BaseRestHandlerTests.java

```diff
@@ -337,4 +343,16 @@ public String getName() {
         }
     }
 
+    public void testReplaceRoutesMethod() throws Exception {
+        List<Route> routes = Arrays.asList(new RestHandler.Route(Method.GET, "/path/test"),
+            new RestHandler.Route(Method.PUT, "/path2/test"));
+        List<ReplacedRoute> replacedRoutes = RestHandler.replaceRoutes(routes, "/prefix", "/deprecatedPrefix");
+
+        for(int i = 0; i < routes.size(); i++) {
+            assertEquals(replacedRoutes.get(i).getPath(), "/prefix" + routes.get(i).getPath());
```

Comment:
order of arguments to `assertEquals` needs to be reversed. (`assertEquals(Object expected, Object actual)`)

Reply:
fixed